### ISSUE TITLE
ppyolo训练自己的数据出现好多nan值

### ISSUE THREAD

**Issue by EnjoyScraping** [PaddlePaddle/PaddleDetection#1374]
`aistudio@jupyter-420458-835152:~/PaddleObjection$ python tools/train.py -c configs/ppyolo/ppyolo.yml --eval

WARNING: Do not have avx core. You may not build with AVX, but AVX is supported on local machine.
 You could build paddle WITH_AVX=ON to get better performance.
The original error is: No module named 'paddle.fluid.core_avx'
W0908 15:54:31.368575   135 init.cc:185] AVX is available, Please re-compile on local machine
2020-09-08 15:54:33,797-INFO: If regularizer of a Parameter has been set by 'fluid.ParamAttr' or 'fluid.WeightNormParamAttr' already. The Regularization[L2Decay, regularization_coeff=0.000500] in Optimizer will not take effect, and it will only be applied to other Parameters!
2020-09-08 15:54:37,236-INFO: places would be ommited when DataLoader is not iterable
W0908 15:54:37.298797   135 device_context.cc:268] Please NOTE: device: 0, CUDA Capability: 70, Driver API Version: 9.2, Runtime API Version: 9.0
W0908 15:54:37.302985   135 device_context.cc:276] device: 0, cuDNN Version: 7.3.
2020-09-08 15:54:40,474-INFO: Downloading ResNet50_vd_ssld_pretrained.tar from https://paddle-imagenet-models-name.bj.bcebos.com/ResNet50_vd_ssld_pretrained.tar
100%|█████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████████| 92837/92837 [00:08<00:00, 11442.72KB/s]
2020-09-08 15:54:48,726-INFO: Decompressing /home/aistudio/.cache/paddle/weights/ResNet50_vd_ssld_pretrained.tar...
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/paddle/fluid/io.py:2052: UserWarning: This list is not set, Because of Paramerter not found in program. There are: fc_0.w_0 fc_0.b_0
  format(" ".join(unused_para_list)))
2020-09-08 15:54:56,175-INFO: places would be ommited when DataLoader is not iterable
2020-09-08 15:55:01,683-INFO: iter: 0, lr: 0.000000, 'loss_xy': '1.265532', 'loss_wh': '3.489594', 'loss_obj': '10467.435547', 'loss_cls': '0.632296', 'loss_iou': '4.285713', 'loss_iou_aware': '7.596504', 'loss': '10484.705078', time: 0.000, eta: 0:00:00
/opt/conda/envs/python35-paddle120-env/lib/python3.7/site-packages/numpy/lib/function_base.py:3405: RuntimeWarning: Invalid value encountered in median
  r = func(a, **kwargs)
2020-09-08 15:55:17,717-INFO: iter: 20, lr: 0.000000, 'loss_xy': 'nan', 'loss_wh': 'nan', 'loss_obj': 'nan', 'loss_cls': 'nan', 'loss_iou': 'nan', 'loss_iou_aware': 'nan', 'loss': 'nan', time: 0.977, eta: 0:02:55
2020-09-08 15:55:30,671-INFO: iter: 40, lr: 0.000001, 'loss_xy': 'nan', 'loss_wh': 'nan', 'loss_obj': 'nan', 'loss_cls': 'nan', 'loss_iou': 'nan', 'loss_iou_aware': 'nan', 'loss': 'nan', time: 0.824, eta: 0:02:11`

这是我的配置文件，我的图片是宽320，高160的：
`architecture: YOLOv3
use_gpu: true
max_iters: 200
log_smooth_window: 100
log_iter: 20
save_dir: output
snapshot_iter: 10000
metric: VOC # COCO
pretrain_weights: https://paddle-imagenet-models-name.bj.bcebos.com/ResNet50_vd_ssld_pretrained.tar
weights: output/ppyolo/model_final
num_classes: 1
use_fine_grained_loss: true
use_ema: true
ema_decay: 0.95

YOLOv3:
  backbone: ResNet
  yolo_head: YOLOv3Head
  use_fine_grained_loss: true

ResNet:
  norm_type: sync_bn
  freeze_at: 0
  freeze_norm: false
  norm_decay: 0.
  depth: 50
  feature_maps: [3, 4, 5]
  variant: d
  dcn_v2_stages: [5]

YOLOv3Head:
  anchor_masks: [[6, 7, 8], [3, 4, 5], [0, 1, 2]]
  anchors: [[10, 13], [16, 30], [33, 23],
            [30, 61], [62, 45], [59, 119],
            [116, 90], [156, 198], [373, 326]]
  norm_decay: 0.
  coord_conv: true
  iou_aware: true
  iou_aware_factor: 0.4
  scale_x_y: 1.05
  spp: true
  yolo_loss: YOLOv3Loss
  nms: MatrixNMS
  drop_block: true

YOLOv3Loss:
  batch_size: 24
  ignore_thresh: 0.7
  scale_x_y: 1.05
  label_smooth: false
  use_fine_grained_loss: true
  iou_loss: IouLoss
  iou_aware_loss: IouAwareLoss

IouLoss:
  loss_weight: 2.5
  max_height: 160
  max_width: 320

IouAwareLoss:
  loss_weight: 1.0
  max_height: 160
  max_width: 320

MatrixNMS:
    background_label: -1
    keep_top_k: 100
    normalized: false
    score_threshold: 0.01
    post_threshold: 0.01

LearningRate:
  base_lr: 0.0001
  schedulers:
  - !PiecewiseDecay
    gamma: 0.1
    milestones:
    - 40000
    - 45000
  - !LinearWarmup
    start_factor: 0.
    steps: 4000

OptimizerBuilder:
  optimizer:
    momentum: 0.9
    type: Momentum
  regularizer:
    factor: 0.0005
    type: L2

_READER_: 'ppyolo_reader.yml'
`

这是ppyolo_reader.yml

`TrainReader:
  inputs_def:
    fields: ['image', 'gt_bbox', 'gt_class', 'gt_score']
    num_max_boxes: 50
  dataset:
    !VOCDataSet
      image_dir: ImageSets
      anno_path: ImageSets/Main/Slider_train.txt
      dataset_dir: dataset/DunSlider-PascalVOC-export
      with_background: false
  sample_transforms:
    - !DecodeImage
      to_rgb: True
      with_mixup: True
    - !MixupImage
      alpha: 1.5
      beta: 1.5
    - !ColorDistort {}
    - !RandomExpand
      fill_value: [123.675, 116.28, 103.53]
    - !RandomCrop {}
    - !RandomFlipImage
      is_normalized: false
    - !NormalizeBox {}
    - !PadBox
      num_max_boxes: 50
    - !BboxXYXY2XYWH {}
  batch_transforms:
  - !RandomShape
    sizes: [320, 352, 384, 416, 448, 480, 512, 544, 576, 608]
    random_inter: True
  - !NormalizeImage
    mean: [0.485, 0.456, 0.406]
    std: [0.229, 0.224, 0.225]
    is_scale: True
    is_channel_first: false
  - !Permute
    to_bgr: false
    channel_first: True
  # Gt2YoloTarget is only used when use_fine_grained_loss set as true,
  # this operator will be deleted automatically if use_fine_grained_loss
  # is set as false
  - !Gt2YoloTarget
    anchor_masks: [[6, 7, 8], [3, 4, 5], [0, 1, 2]]
    anchors: [[10, 13], [16, 30], [33, 23],
              [30, 61], [62, 45], [59, 119],
              [116, 90], [156, 198], [373, 326]]
    downsample_ratios: [32, 16, 8]
  batch_size: 24
  shuffle: true
  mixup_epoch: 200
  drop_last: true
  worker_num: 8
  bufsize: 4
  use_process: true

EvalReader:
  inputs_def:
    fields: ['image', 'im_size', 'im_id']
    num_max_boxes: 50
  dataset:
    !VOCDataSet
      image_dir: ImageSets
      anno_path: ImageSets/Main/Slider_val.txt
      dataset_dir: dataset/DunSlider-PascalVOC-export
      with_background: false
  sample_transforms:
    - !DecodeImage
      to_rgb: True
    - !ResizeImage
      target_size: 608
      interp: 2
    - !NormalizeImage
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]
      is_scale: True
      is_channel_first: false
    - !PadBox
      num_max_boxes: 50
    - !Permute
      to_bgr: false
      channel_first: True
  batch_size: 8
  drop_empty: false
  worker_num: 8
  bufsize: 4

TestReader:
  inputs_def:
    image_shape: [3, 320, 160]
    fields: ['image', 'im_size', 'im_id']
  dataset:
    !ImageFolder
      anno_path: annotations/instances_val2017.json
      with_background: false
  sample_transforms:
    - !DecodeImage
      to_rgb: True
    - !ResizeImage
      target_size: 608
      interp: 2
    - !NormalizeImage
      mean: [0.485, 0.456, 0.406]
      std: [0.229, 0.224, 0.225]
      is_scale: True
      is_channel_first: false
    - !Permute
      to_bgr: false
      channel_first: True
  batch_size: 1
`
